### DISCUSSION TITLE
OMNeT++ 6.0.3 released

### DISCUSSION THREAD

**Discussion by rhornig** [omnetpp/omnetpp#1111] [Announcements]
This is a maintenance release, with several performance improvements in the simulation kernel and further adjustments.

Documentation:
  - Updated install instructions for macOS to support installing on aarch64 based computers using Homebrew.
   
IDE:
  - Updated to Eclipse 4.30, CDT 11.4, Pydev 11.0
  - IDE is now fully supported on macOS/aarch64
  - Model installation dialogs: If the IDE was installed using opp_env, tell the user to install the model with opp_env, too.
  
Some reported issues were also [fixed](https://github.com/omnetpp/omnetpp/issues?q=is%3Aissue+milestone%3A6.0.3+is%3Aclosed).

<hr /><em>This discussion was created from the release <a href='https://github.com/omnetpp/omnetpp/releases/tag/omnetpp-6.0.3'>OMNeT++ 6.0.3</a>.</em>

**Comment by haug-den-lucas** [comment]
Would it be possible to also create a docker container for the new release?

Additionally, I think it would be great if GitHub Actions would be used to automatically create docker containers on new releases.

**Reply by rhornig** [reply]
Yep. I will do 6.0.3 image. I'm somewhat new to github actions but that is something I want to do eventually.
